### PR TITLE
[ja]Update kubeadm installation instructions in Japanese documentation

### PR DESCRIPTION
**Pull request by aoki-taquan**
issue: #44495.
Due to operational errors in the previous PR, it ended up in an awkward state. So I re-created the PR.
The original PR is PR: #44496.

As suggested in the previous PR, I have made updates beyond fixing the problem, specifically revising the entire section related to Debian-based distributions.

### CODE REVIEW

**Comment by aoki-taquan** [comment]
@t-inu
I made the changes as you suggested. Is everything OK?

**Comment by Okabe-Junya** [comment]
The target pages on the deploy preview: 

- https://deploy-preview-44950--kubernetes-io-main-staging.netlify.app/ja/docs/setup/production-environment/tools/kubeadm/install-kubeadm/

File: content/ja/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md [hunk 1]
```diff
@@ -126,19 +126,27 @@ kubeadmは`kubelet`や`kubectl`をインストールまたは管理**しない**
 
    ```shell
    sudo apt-get update
+   # apt-transport-httpsはダミーパッケージの可能性があります。その場合、そのパッケージはスキップできます
    sudo apt-get install -y apt-transport-https ca-certificates curl gpg
    ```
 
-2. Google Cloudの公開鍵をダウンロードします:
+2. Kubernetesパッケージリポジトリの公開署名キーをダウンロードします。すべてのリポジトリに同じ署名キーが使用されるため、URL内のバージョンは無視できます:
```
Comment (Okabe-Junya) [inline content/ja/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md:133]:
```suggestion
2. Kubernetesパッケージリポジトリーの公開署名キーをダウンロードします。すべてのリポジトリーに同じ署名キーが使用されるため、URL内のバージョンは無視できます:
```

非常に微妙なのですが、ルール上は「リポジトリー」が正しそうですね。。

FYI: 

- Slack: https://kubernetes.slack.com/archives/CAG2M83S8/p1574910937227000
- https://kubernetes.io/ja/docs/contribute/localization/#単語末尾に長音記号-ー-を付けるかどうか

File: content/ja/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md [hunk 2]
```diff
@@ -126,19 +126,27 @@ kubeadmは`kubelet`や`kubectl`をインストールまたは管理**しない**
...
+   # sudo mkdir -p -m 755 /etc/apt/keyrings
+   curl -fsSL https://pkgs.k8s.io/core:/stable:/{{< param "version" >}}/deb/Release.key | sudo gpg --dearmor -o /etc/apt/keyrings/kubernetes-apt-keyring.gpg
    ```
 
-3. Kubernetesの`apt`リポジトリを追加します:
+{{< note >}}
+Debian 12とUbuntu 22.04より古いリリースでは、`/etc/apt/keyrings`フォルダーはデフォルトでは存在しないため、curlコマンドの前に作成する必要があります。
+{{< /note >}}
+
+3. 適切なKubernetes `apt`リポジトリを追加します。このリポジトリには、Kubernetes {{< skew currentVersion >}}用のパッケージのみがあることに注意してください。他のKubernetesマイナーバージョンの場合は、目的のマイナーバージョンに一致するようにURL内のKubernetesマイナーバージョンを変更する必要があります(インストールする予定のKubernetesバージョンのドキュメントも読んでください):
```
Comment (Okabe-Junya) [inline content/ja/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md:145]:
```suggestion
3. 適切なKubernetes `apt`リポジトリーを追加します。このリポジトリーには、Kubernetes {{< skew currentVersion >}}用のパッケージのみがあることに注意してください。他のKubernetesマイナーバージョンの場合は、目的のマイナーバージョンに一致するようにURL内のKubernetesマイナーバージョンを変更する必要があります(インストールする予定のKubernetesバージョンのドキュメントも読んでください):
```

ditto

> 非常に微妙なのですが、ルール上は「リポジトリー」が正しそうですね。。

File: content/ja/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md (hunk 1 above)
Comment (Okabe-Junya) [reply content/ja/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md:133]: @nasa9084 

一応確認ですが、「リポジトリー」表記に統一する形で良いでしょうか。。？？

**Review by Okabe-Junya — COMMENTED** [review]
Nice catch!! Thank you!!

I have made a few comments, but they are nits :)

**Comment by Okabe-Junya** [comment]
レビューに書けませんでしたが、修正する場合はL125も合わせて変更する必要がありそうですね :)

File: content/ja/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md (hunk 1 above)
Comment (Okabe-Junya) [reply content/ja/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md:133]: PTAL @aoki-taquan

**Comment by Okabe-Junya** [comment]
/lgtm

Thank you!!

**Comment by Okabe-Junya** [comment]
PTAL @aoki-taquan :)

> Please delete lines 159-162. [#44496 (comment)](https://github.com/kubernetes/website/pull/44496#discussion_r1466586232)

/remove-lgtm

**Comment by Okabe-Junya** [comment]
Nice catch!! Thank you, LGTM!!

/lgtm

**Comment by aoki-taquan** [comment]
> Please delete lines 159-162. [#44496 (comment)](https://github.com/kubernetes/website/pull/44496#discussion_r1466586232)

Thanks. I fixed it.

**Comment by nasa9084** [comment]
/approve
thank you for your contributing~

**Pull request merged**